api/routes/resource.routes.js

```diff
@@ -67,7 +112,40 @@ module.exports = (app) => {
       await resourceUtil.update(req.params, req.body);
       res.json({});
     },
-    { owner: ['_id', 'fillme'], mod: ['_id', 'fillme'] },
+    {
+      owner: [
+        'name',
+        'desc',
+        'type',
+        'topics',
+        'path',
+        'downloadURL',
+        'modifiedDate',
+        'trustIndexCategories',
+        'keywords',
+        'organizations',
+        'files',
+        'creator',
+        'reviewsRemaining',
+        '_id',
+      ],
+      mod: [
+        'name',
+        'desc',
+        'type',
+        'topics',
+        'path',
+        'downloadURL',
+        'modifiedDate',
+        'trustIndexCategories',
+        'keywords',
+        'organizations',
+        'files',
+        'creator',
+        'reviewsRemaining',
+        '_id',
+      ],
```

Comment:
can you make this (or part of it) a global constant in resource util (eg resource update fields) and then just ref that here. This is pretty verbose. Same comment for the other places where there are more than like 5 fields that are repeated.